{"name": "test_multiple_login_page_success[invalid_user4-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user4'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753501197503, "stop": 1753501198483, "uuid": "ab4e18a8-490a-4154-b511-95e899b86384", "historyId": "8169f900955b8e47d48e3638b02b295f", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "49910-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}